Source/Core/PixelFormat.js

```diff
@@ -47,6 +47,14 @@ var PixelFormat = {
    */
   RGBA: WebGLConstants.RGBA,
 
+  /**
+   * A pixel format containing unsigned normalized red, green, blue, and alpha channels.
+   *
+   * @type {Number}
+   * @constant
+   */
+  RGBA_UNORM: WebGLConstants.UNSIGNED_NORMALIZED,
```

Comment:
@lilleyse I'm pretty certain this is the wrong WebGL constant to use, but looking at the docs [here](https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Constants) I couldn't find one that matches the 8-bit UNORM format of the IBL sampler... SNORM is the closest I see. Any thoughts here?

Reply:
`RGBA` is implicitly UNORM when paired with `PixelDatatype.UNSIGNED_BYTE` so an additional pixel format does not have to be added

Reply:
`RGBA` is implicitly UNORM when paired with `PixelDatatype.UNSIGNED_BYTE` so an additional pixel format does not have to be added